WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -276,11 +276,11 @@ private struct CartScrollViewContent: View {
                     .onAppear {
                         updateItemImageVisibility(cartListWidth: geometry.size.width)
                     }
-                    .onChange(of: geometry.size.width) {
-                        updateItemImageVisibility(cartListWidth: $0)
+                    .onChange(of: geometry.size.width) { _, newValue in
```

Comment:
This is no longer something we can do, since we're using the new value inside the closure,  we need to be declare it explicitly in `.onChange`

Reply:
Can't you use $1 instead? This is fine though.

Reply:
Oh, ... that's right 🤦